src/fontra/views/fontinfo/panel-development-status-definitions.js

```diff
@@ -186,24 +190,32 @@ class StatusDefinitionBox extends HTMLElement {
     const statusDefinitions =
       this.fontController.customData["fontra.sourceStatusFieldDefinitions"];
     if (statusDefinitions.some((statusDef) => statusDef.value == statusDefValue)) {
-      errorMessage = `“${statusDefValue}” exists already, please use a different value.`;
+      errorMessage = translate("warning.entry-exists", statusDefValue);
     }
 
     if (!Number.isInteger(statusDefValue) || statusDefValue < 0) {
-      errorMessage = "Value must be a positive number.";
+      errorMessage = translate(
+        "development-status-definitions.warning.positive-number"
+      );
     }
 
     if (errorMessage) {
-      message(`Can’t edit status definition value`, errorMessage);
+      message(
+        translate(
+          "development-status-definitions.dialog.cannot-edit-status-definition.title"
+        ),
+        errorMessage
+      );
       return false;
     }
     return true;
   }
 
-  replaceStatusDef(newStatusDef, undoLabel, statusIndex = this.statusIndex) {
+  replaceStatusDef(newStatusDef) {
+    const undoLabel = translate("development-status-definitions.undo.change-color");
```

Comment:
This function is used to change the color *and* the label, so now the label is wrong for the label case.

However, it's quite irresponsible you spent time on this to begin with, as these undo labels aren't even visible anywhere: we only have command-Z and command-shift-Z. No menu bar, no context menu.

Now that this code is there, I think it's fine if both calls to `replaceStatusDef()` share their undo message, it just needs to be generic, and not mention "color" or "label", also not in the key.

Reply:
I'm sorry. I changed it and updated it with a more generic key and translation.